templates/zerver/integrations.html

```diff
@@ -1009,6 +1009,48 @@
 
     </div>
 
+    <div id="hellosign" class="integration-instructions">
+
+      <p>
+        Configuring the HelloSign integration is easy, first, create a stream
+        (we recommend naming it <code>hellosign</code>), if no stream is supplied,
+        the default is hellosign if it exists. This does not mean you don't
```

Comment:
* This is a very long sentence. Would all full stops after `easy`and the brackets.

* `if no stream name is provided` 

* "hellosign" in quotation marks for the default

* I'd say `create a stream` instead of establish.

* I think the last sentence could be a bit unclear, simply say `You need to have a stream for the HelloSign bot to post messages to` or sth similar.